server/schema.graphql

```diff
@@ -1,3 +1,7 @@
 type Query {
     info: String!
-}
\ No newline at end of file
+    feed: String!
+    trainFeed(train:String): String
+    arrivalTimes(stationId: String, train: String, direction: String): String
```

Comment:
we need to make these match the actual data types right? I'm happy to help with that.